Test/Unit/Block/JsTest.php

```diff
@@ -241,4 +247,39 @@ public function setSandboxMode($value = true)
             ->method('isSandboxModeSet')
             ->will($this->returnValue($value));
     }
+    
+    /**
+     * Call protected/private method of a class.
+     *
+     * @param $object
+     * @param $methodName
+     * @param array $parameters
+     * @return mixed
+     */
+    public function invokeMethod(&$object, $methodName, array $parameters = [])
+    {
+        $reflection = new \ReflectionClass(get_class($object));
+        $method = $reflection->getMethod($methodName);
+        $method->setAccessible(true);
+
+        return $method->invokeArgs($object, $parameters);
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function testGetQuoteIsVirtual()
```

Comment:
again, I think this test doesn't test anything interesting. tested method is basically one-line getter so having test doesn't provide much value here.